etc/rules/msauth_rules.xml

```diff
@@ -876,7 +876,13 @@
     <description>User Logoff Exchange.</description>
   </rule>
   
-  
+    <rule id="18262" level="3">
+    <if_sid>18107</if_sid>
+    <id>^4624$</id>
+    <match>Logon Type:   2</match>
```

Comment:
yes, it's 3 spaces, default by microsoft win event. And i see windows logon event only maxium logon type =11